src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -727,6 +727,16 @@ def prepareLocal(self, dagSpecs, info, kw, inputFiles, subdags):
         finally:
             tf.close()
 
+        # also upload InputFiles.tar.gz to s3
+        # Wa: Now (2024) I am still not sure if we need to poll uploaded file
```

Comment:
well, here we upload from a CERN node a <2MB file, not a large user sandbox from a remote client, either S3 is down, or it will be lightening fast.

Reply:
But sandbox should finish uploading (from client) before reaching this line, no?

Or maybe I misread the code :thinking:

Reply:
we agree to
   *add a call to the new checkS3Object() at the end of uploadToS3*

Reply:
all done inb https://github.com/dmwm/CRABServer/pull/8740